src/Recommendations.jsx

```diff
@@ -25,12 +26,24 @@ function Recommendations() {
       />
     ));
   function fetchRecommendations() {
-    fetch(`http://localhost:3000/ml/fetchRecs?userId=${currentUser.uid}`)
+    fetch(
+      `http://localhost:3000/ml/fetchRecs?userId=${currentUser.uid}&age=${age}`
+    )
       .then((response) => response.json())
       .then((data) => {
         setRecommendations(data);
       });
   }
+  useEffect(() => {
```

Comment:
Can age be included by default in User to avoid needing to make this additional call?

Reply:
I changed it in the latest iteration "PR 26" so that I just fetch the user as a whole and then access age, gender, and occupation, from within that. Is that what you meant?